Applications/Calendar/CalendarWidget.cpp

```diff
@@ -0,0 +1,169 @@
+#include "CalendarWidget.h"
+#include "AddEventDialog.h"
+#include <LibCore/DateTime.h>
+#include <LibGUI/BoxLayout.h>
+#include <LibGUI/Button.h>
+#include <LibGUI/Painter.h>
+#include <LibGUI/Window.h>
+#include <LibGfx/Font.h>
+
+CalendarWidget::CalendarWidget()
+{
+    set_fill_with_background_color(true);
+    m_calendar = new Calendar(Calendar::get_year(), Calendar::get_month());
+
+    m_selected_date_label = add<GUI::Label>();
+    m_selected_date_label->set_relative_rect(10, 13, 100, 25);
+    m_selected_date_label->set_text_alignment(Gfx::TextAlignment::CenterLeft);
+    //TODO: Change text to a bigger size.
+    m_selected_date_label->set_font(Gfx::Font::default_bold_font());
+    m_selected_date_label->set_text(m_calendar->get_selected_date_text());
+
+    m_prev_month_button = add<GUI::Button>();
+    m_prev_month_button->set_text("<");
+    m_prev_month_button->set_font(Gfx::Font::default_bold_font());
+    m_prev_month_button->set_relative_rect(90, 5, 40, 40);
+    m_prev_month_button->on_click = [&] {
```

Comment:
While it is safe in this context, I would advise against capturing `[&]` like this when installing persistent callbacks.
`&` will capture references to everything on the current stack, all of which will be dangling by the time we leave the `Calendar` constructor. My recommendation is to capture `[this]`, which will capture nothing but the raw `this` pointer.